src/webui/api/synccontroller.cpp

```diff
@@ -462,7 +462,7 @@ void SyncController::maindataAction()
     {
         const BitTorrent::InfoHash torrentHash = torrent->hash();
 
-        QVariantMap map = serialize(*torrent);
+        auto map = serialize(*torrent).toVariantMap();
```

Comment:
`QVariantMap` is still used because there are (a lot of) code that will require this type.

Reply:
Then does it even make sense to add another transformation (`JSON - Variant -JSON` instead of current `Variant -JSON`)? I assumed we could get rid of one instead...

Reply:
I would still consider https://github.com/qbittorrent/qBittorrent/pull/13969#discussion_r542284198.